FIND-252: long filter collection scaffold

TNA-Allan opened the pull request:
# Pull Request

Ticket URL: [FIND-252](https://national-archives.atlassian.net/browse/FIND-252)

## About these changes

Add a human-readable description here

## How to check these changes

Where possible, provide guidance to help your reviewer

## Before assigning to reviewer, please make sure you have

- Checked things thoroughly before handing over to reviewer
- Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- Ensured that PR includes only commits relevant to the ticket
- Waited for all CI jobs to pass before requesting a review
- Added/updated tests and documentation where relevant

TNA-Allan commented on app/search/buckets.py:
```diff
@@ -1,20 +1,35 @@
-    CLOSURE = "closure"
+    LEVEL = ("level", "")
+    COLLECTION = ("collection", "longCollections")
+    HELD_BY = ("heldBy", "")  # TODO: long_aggs in another PR change to held_by
```
Enum value isn't changing. Removed change to.

TNA-Allan commented on app/search/views.py:
```diff
@@ -178,6 +182,18 @@ def setup(self, request: HttpRequest, *args, **kwargs) -> None:
         # create two separate forms for TNA and NonTNA with different fields
         if form_kwargs.get("data").get("group") == BucketKeys.TNA.value:
             self.form = CatalogueSearchTnaForm(**form_kwargs)
+
+            # ensure only single value is bound to ChoiceFields
+            for field_name, field in self.form.fields.items():
+                if isinstance(field, ChoiceField):
+                    if len(form_kwargs.get("data").getlist(field_name)) > 1:
```
`self.get_form_kwargs()` ensures `form_kwargs.get('data')` will not return None.

JohnHeeryTNA commented:
Tested branch locally and approving from a FE perspective.

JohnHeeryTNA reviewed (approved):
Tested Locally and approving changes from FE perspective

TNA-Allan commented:
Hi @Firdaussi , I've addressed the feedback — could you please take another look and let me know if it’s good to go from the BE perspective, or if there's anything else?
